seir/scripts/cli.py

```diff
@@ -40,12 +50,17 @@ def main():
     results = model.evaluate_solution(time)
 
     # Save data
-    results.to_csv('outfile.csv')
+    results.to_csv(output_file)
 
     # Visualize the results
     visualize_seir_computation(results,
-                               show_individual_compartments = True)
+                               show_individual_compartments=True)
 
 
 if __name__ == '__main__':
-    main()
+    parser = argparse.ArgumentParser(description='Modeling epidemics using the SEIR model')
+    parser.add_argument('config_file', type=str, help='File path to config ini file')
+    parser.add_argument('-c', dest="contacts_matrix_file", type=str, help='File path to contact matrix file')
+    parser.add_argument('-o', dest="output_file", default="outfile.csv", type=str, help='Output file name')
+    args = parser.parse_args(sys.argv[1:])
```

Comment:
`sys.argv[1:]`

Should not be necessary, no?

Reply:
True :)